helix-core/src/test/java/org/apache/helix/controller/strategy/crushMapping/TestCardDealingAdjustmentAlgorithmV2.java

```diff
@@ -187,9 +189,11 @@ public static Object[][] stableComputingVerification() {
     };
   }
 
-  @Test(description = "Compute mapping multiple times, the mapping of each time should be same", dataProvider = "stableComputingVerification")
+  @Test(description = "Compute mapping multiple times, the mapping of each time should be same", dataProvider = "stableComputingVerification",
+      dependsOnMethods = "testAlgorithmConstructor")
   public void testStableComputeMappingForMultipleTimes(int replica, int repeatTimes, int seed,
       boolean isEvennessPreferred) {
+    System.out.println("START TestCardDealingAdjustmentAlgorithmV2.testStableComputeMappingForMultipleTimes");
```

Comment:
Nit: we could use TestHelper to get method name for print statements

Reply:
Thanks for the suggestion. I think normally we just use the plan text test func name to reduce string append.